app/src/org/commcare/android/javarosa/IntentCallout.java

```diff
@@ -298,32 +307,48 @@ private void storePointerToFileResponse(TreeReference ref, String responseValue,
             return;
         }
 
-        //Otherwise, grab that file
-        File src = new File(responseValue);
-        if (!src.exists()) {
-            //TODO: How hard should we be failing here?
-            Log.w(TAG, "CommCare received a link to a file at " + src.toString() + " to be included in the form, but it was not present on the phone!");
+        try {
+            File localCopy = copyFileToLocalDirectory(responseValue, destinationFile);
+            formDef.setValue(new StringData(localCopy.getName()), ref);
+            return;
+        } catch (FileNotFoundException fnfe) {
+            Log.w(TAG, "CommCare received a link to a file at " + responseValue + " to be included in the form, but it was not present on the phone!");
+
+            //Wipe out any reference that exists
+            formDef.setValue(null, ref);
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+            Log.e(TAG,"CommCare failed to copy a binary input from an intent callout from: " + responseValue);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/a72c848e609bc036eec2b6ba409562525ed25893/app/src/org/commcare/android/javarosa/IntentCallout.java#L321"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>